www/src/components/marketplace/Marketplace.js

```diff
@@ -53,32 +53,30 @@ function Marketplace({ installed }) {
         </Flex>
       </Flex>
       <Flex
-        id="1"
         mt={1}
         flexGrow={1}
-        maxHeight="100%"
+        overflow="hidden"
       >
-        <Flex
-          direction="column"
+        <MarketplaceRepositories
+          installed={installed}
           flexGrow={1}
-        >
-          <MarketplaceRepositories
-            installed={installed}
-          />
-        </Flex>
+        />
         <Div
           ml={areFiltersOpen ? 2 : 0}
+          mr={areFiltersOpen ? 0 : `-${256 - 32}px`}
+          transform={areFiltersOpen ? 'translateX(0)' : 'translateX(100%)'}
+          opacity={areFiltersOpen ? 1 : 0}
           flexShrink={0}
           position="sticky"
           top={0}
-          left={0}
-          width={areFiltersOpen ? 256 - 32 : 0}
-          height={`calc(100% - ${32}px)`}
+          right={0}
+          width={256 - 32}
```

Comment:
is there a reason why we don't just make thse constants?

Reply:
I will